### DISCUSSION TITLE
why I get ver. 3.3.7 via git, when I see the latest is 3.3.6 ?

### DISCUSSION THREAD

**Discussion by jancovic** [netbox-community/netbox#10790] [General]
1) Why I get ver. 3.3.7 dev  via git, when I see on the main page is the latest is 3.3.6 ?
https://github.com/netbox-community/netbox.git
Its probably due to develop branch... so what branch is stable ?

2) From downloaded package, where can I see the number release. ?

3) from installed app dir, where can I see the version number ?

**Comment by candlerb** [comment]
The "master" branch is stable and has the current released version.

The downloaded releases are each itemised with their version numbers, see https://github.com/netbox-community/netbox/releases

If you're asking where the version number is stored within the code, then

```
# grep -R 3\\.3\\.6 /opt/netbox/netbox | cut -c1-80
/opt/netbox/netbox/netbox/settings.py:VERSION = '3.3.6'
```

**Comment by jancovic** [comment]
OK, when I already migrated to 3.3.7 devel, can I migrate back to 3.3.6, or it is not good idea migrate data to lower ver. ?

**Reply by kkthxbye-code** [reply]
There has been no changes in 3.3.7-dev yet, so it should be fine to migrate back to 3.3.6.
